### ISSUE TITLE
gyrojet

### ISSUE THREAD

**Issue by ATPGaming** [studio-minus/ppg-bugs-and-requests#2148]
**Feature description**
add the unique and infamous gyrojet pistol, a lightweight handgun that shoots a small rocket rather than bullets. just like real life though, the rocket is highly inaccurate due to its size

**Feature purpose**
a pistol that shoots a rocket would be awesome, especially since we don't have very many rocket weapons

**Additional context**
![image](https://user-images.githubusercontent.com/109692890/190831866-8d2e99ce-5e0f-427c-8572-7b23afb692c2.png)
![image](https://user-images.githubusercontent.com/109692890/190831922-488b7e6b-abf7-4b23-b61f-ff2983051e28.png)
I didn't find any mods of gyrojet on the workshop, so this can't be added by a mod... yet.

**Comment by DindinYT37** [comment]
Of course it can be added by a mod, it's just nobody makes it because it's too niche. You'd have a better chance at getting this added asking a mod-maker.

**Comment by Drogo-man** [comment]
Technically everything can just be modded in, but u don't think you know just how limited the mod api is, and most people just don't bother. I think some things, like this, are best added in vanilla.

**Comment by DindinYT37** [comment]
> Technically everything can just be modded in, but u don't think you know just how limited the mod api is, and most people just don't bother. I think some things, like this, are best added in vanilla.

I know how limited the modding API is. The problem with your argument here is, that this could even be done by simple putting a re-skin on the rocket launcher.

**Comment by Drogo-man** [comment]
But that would just be a thing launcher that looks like a gyrojet, not a gyrojet.

**Comment by DindinYT37** [comment]
> But that would just be a thing launcher that looks like a gyrojet, not a gyrojet.

Yeah, a weapon that looks like a gyrojet and does what a gyrojet does.

**Comment by Drogo-man** [comment]
But still, why can't it just be added in vanilla? It's an original thing.

**Comment by DindinYT37** [comment]
> But still, why can't it just be added in vanilla? It's an original thing.

I don't say it can't be added in vanilla - I just say it can also be added by mods since he said "so this can't be added by a mod... yet".

**Comment by Drogo-man** [comment]
What about the fact that modders aren't the devs.

**Comment by DindinYT37** [comment]
> What about the fact that modders aren't the devs.

Not related to this discussion at all? I was simply explaining that it can indeed be added by a mod, while he said it can't. You're making this unnecessarily complicated.

**Comment by ATPGaming** [comment]
WHAT HAVE I DONE

**Comment by ATPGaming** [comment]
And I thought I was the only one who did this

**Comment by DindinYT37** [comment]
> And I thought I was the only one who did this

Did... what?

**Comment by ATPGaming** [comment]
Started an argument

**Comment by Drogo-man** [comment]
This is what i mean, people can't have ideas because they can be modded in, or the template was wrong, or there is a similar request, or the idea is just stupid.

Bruh.

**Comment by DindinYT37** [comment]
> This is what i mean, people can't have ideas because they can be modded in, or the template was wrong, or there is a similar request, or the idea is just stupid.
> 
> Bruh.

They can have an idea, but that doesn't mean it's a good idea. Some things can be modded in but would still be nice to be in vanilla, though this request was just another gun and is also similar basically a grenade launcher. Nobody said requests can't be accepted because the template is wrong or they're similar to another, and never will anyone do.

..and if the idea is just stupid then it doesn't really deserve to be accepted in the first place?

**Comment by WWWOWWWPL** [comment]
nice idea
